frontend/src/app/register/register.component.html

```diff
@@ -0,0 +1,50 @@
+<h2>Rejestracja w Volontulo</h2>
```

Comment:
We don't have an explicit rule here, but I would prefer 2-char intendation in html tempaltes.